locale/it_IT/admin.xml

```diff
@@ -39,6 +39,7 @@
 	<message key="admin.languages.reload">Ricarica localizzazione</message>
 	<message key="admin.languages.uninstall">Disinstalla localizzazione</message>
 	<message key="admin.phpInfo">Informazioni dettagliate su PHP</message>
+	<message key="admin.scheduledTask.subscriptionExpiryReminder">Subscription expiry reminder</message>
```

Comment:
The line was doubled for error. The file already contains the key `admin.scheduledTask.subscriptionExpiryReminder` stranslated into Italian. I'll remove this line.